server/src/infra/repositories/asset.repository.ts

```diff
@@ -250,7 +250,7 @@ export class AssetRepository implements IAssetRepository {
     builder = searchAssetBuilder(builder, options);
     builder.orderBy('asset.fileCreatedAt', options.orderDirection ?? 'DESC');
```

Comment:
this is only used for this getAll endpoint for mobile and it should be sorted by createdAt not fileCreatedAt, if you want to avoid the list changing on you.